roles/slurm_management/tasks/main.yml

```diff
@@ -100,20 +100,20 @@
   become: true
 
 - name: 'Create a database for Slurm accounting.'
-  mysql_db:
+  community.mysql.mysql_db:
     login_host: 'localhost'
     login_user: 'root'
     login_password: "{{ MYSQL_ROOT_PASSWORD }}"
     login_unix_socket: "/var/lib/mysql/mysql.sock"
     name: "{{ slurm_database_name }}"
     state: 'present'
-  no_log: true
+  no_log: false
```

Comment:
This will log also the password, which is Ok for debugging, but a security risk for production.

Reply:
Correct - I forgot to change it back.